### DISCUSSION TITLE
can someone help me with this code?

### DISCUSSION THREAD

**Discussion by MJmcMaster** [nesbox/TIC-80#1781] [Q&A]
I'm trying to run the game to see what it looks like for now, and i get this message:

[string "--title: (ok this is the title skip this part). . ."] :10: <name> expected near ')'

and i can't find out waht it means, i've already fixed some other ones but i can't seem to fix this one. here is some code for the game:

t=0
x=96
y=24

function) tic()

can anyone help me with that

yes that function) tic() part is the 10th line

**Comment by paul59** [comment]
Line 10 should read:

function TIC()

You have a misplaced bracket after the word 'function' (and note the uppercase 'TIC')
